src/StatusQ/Popups/statusModal/StatusImageWithTitle.qml

```diff
@@ -64,12 +66,35 @@ Row {
                 width: statusImageWithTitle.image.width
                 height: statusImageWithTitle.image.height
                 color: statusImageWithTitle.image.isIdenticon ?
-                    Theme.palette.statusRoundedImage.backgroundColor :
-                    "transparent"
+                           Theme.palette.statusRoundedImage.backgroundColor :
+                           "transparent"
                 border.width: statusImageWithTitle.image.isIdenticon ? 1 : 0
                 border.color: Theme.palette.directColor7
                 showLoadingIndicator: true
             }
+
+            StatusRoundButton {
+                type: StatusRoundButton.Type.Secondary
+
+                anchors.right: parent.right
+                anchors.bottom: parent.bottom
+                anchors.bottomMargin: -3
+
+                width: 15
+                height: 15
```

Comment:
I believe this has to be bigger though. Looking at the figma, the button is 16x16 and also, you can see in the UI that it's super small:
<img width="183" alt="Screenshot 2021-10-13 at 14 03 52" src="https://user-images.githubusercontent.com/445106/137129018-479dd380-481c-43c7-9273-76c49f5cdb00.png">

Reply:
Keep in mind that the `border.width` wont affect the overall size

Reply:
Also, it looks like it isn't positioned correctly. It's hanging "off" a but too much